### DISCUSSION TITLE
Adding an icon to a textbox: is there a better way to do this?

### DISCUSSION THREAD

**Discussion by AndreSteenbergen** [AvaloniaUI/Avalonia#12906] [Q&A] [answered]
It is only for the login page I am adding some icons to textboxes, but doing it like this feels a bit dirty.

![image](https://github.com/AvaloniaUI/Avalonia/assets/7676790/4be35429-eae0-4e82-98a4-8a9d05fd0a01)

**Comment by timunie** [comment, marked as answer]
Use `TextBox.InnerLeftContent` :-)

**Reply by AndreSteenbergen** [reply]
Thanks that makes sence
